zulip_bots/generate_manifest.py

```diff
@@ -68,7 +72,7 @@ def get_docs():
     return doc_paths
 
 def get_assets():
-    # type: () -> List[str]
+    # type: () -> Iterator[str]
```

Comment:
Since `map()` returns a list, can we do `from typing import List` and keep the List types? Also for the other occurrences in this file.

Reply:
That's how it was originally. I changed it to avoid this error: `Incompatible return value type (got Iterator[str], expected List[str])`.

@neiljp FYI :)